app/components/chat.tsx

```
@@ -390,6 +419,211 @@ export const Chat = ({
+    if (!triggerRun || aiParts.length === 0) {
+      if (!triggerRun) setTriggerAssistantMessage(null);
+      return;
+    }
+
+    const generation = ++aiPartsGenerationRef.current;
+
+    type Chunk = import("ai").UIMessageChunk;
+    const stream = new ReadableStream<Chunk>({
+      start(controller) {
+        try {
+          (aiParts as Chunk[]).forEach((chunk: Chunk) =>
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)